types.bal

```diff
@@ -155,40 +147,47 @@ public isolated function mapToExperimentDataResponse(database:ExperimentDataFull
     }
     if (inputFor != ()) {
         dataMapped.inputFor = inputFor;
-        dataMapped.inputForLinks = from var step in inputFor select string `/experiments/${data.experimentId}/timeline/${step}`;
+        dataMapped.inputForLinks = from var step in inputFor
+            select string `/experiments/${data.experimentId}/timeline/${step}`;
     }
     return dataMapped;
 }
 
-
 ////////////////////////////////////////////////////////////////////////////////
 // Timeline ////////////////////////////////////////////////////////////////////
 ////////////////////////////////////////////////////////////////////////////////
 
 public type TimelineStepPost record {|
     string resultLocation;
     string[] inputData;
-
     string processorName;
-    string? processorVersion=();
-    string? processorLocation=();
+    string? processorVersion = ();
+    string? processorLocation = ();
     string? parameters;
-    string parametersContentType=mime:APPLICATION_FORM_URLENCODED;
-    string? parametersDescriptionLocation=();
-|};
+    string parametersContentType = mime:APPLICATION_FORM_URLENCODED;
+    string? parametersDescriptionLocation = ();
+    int? progressValue = ();
+    int? progressStart = ();
+    int? progressTarget = ();
+    string? progressUnit = ();
```

Comment:
see comment in API. probably no change needed

Reply:
changed to *database:Progress